share/revng/test/configuration/revng-qa/architectures.yml

```diff
@@ -87,11 +98,28 @@ tags:
       QEMU_NAME: s390x
       COMMON_CFLAGS:
         - -DTARGET_s390x
+      CLANG_CFLAGS:
+        - --target=systemz
   - name: aarch64
     implies: [cross-compiler]
     variables:
       TRIPLE: aarch64-unknown-linux-musl-
       POINTER_SIZE: "64"
       QEMU_NAME: aarch64
+      APPLE_NAME: arm64
       COMMON_CFLAGS:
         - -DTARGET_aarch64
+      CLANG_CFLAGS:
+        - --target=aarch64
+
+  - name: apple-x86-64
+    implies: [x86-64, clang-only]
```

Comment:
1. Does `x86-64` imply `linux`? `apple-x86-64` should not imply `linux`. I'd like an `apple` tag (or `darwin`).

Reply:
After the first big msvc PR went in, these no longer imply linux. Only `cross-compiler`.